### PR TITLE
More small changes to styling, including addition of sidebar data.

### PR DESCRIPTION
**Pull request by ganewman**
### What is a quick description of the change?

This PR makes many small additions to the styling:

- Include the Material "How to Vote" icon within the website header.
- Make the website header clickable (redirects to landing page).
- Use `ng-show` to hide empty fields on the candidate page.
- Add styling to lists on the directory and candidate pages.
- Set a maximum width on the sidebar to prevent overlap with page content.
- Dynamically add links to election sections to the sidebar on the directory page.

### Is this fixing an issue?

none

### Are there more details that are relevant?

none

### Check lists (check `x` in `[ ]` of list items)

- [  ] Test written/updating
- [X] Tests passing
- [X] Coding style (indentation, etc)

Please explain why any are not present, if any.

### CODE REVIEW

File: project/src/main/webapp/candidate-page/candidate-page.template.html [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
+    <li ng-show="officialInfo.email">Email: {{officialInfo.email}}</li>
+    <li ng-show="officialInfo.phoneNumber">
+        Phone number: {{officialInfo.phoneNumber}}</li>
   </ul>
   <p ng-if="officialInfo.isIncumbent"> {{officialInfo.candidateName}}
         currently holds office as {{officialInfo.position}}</p>
-  <p>Click <a href={{officialInfo.website}}>here </a>to be redirected
-      to {{officialInfo.candidateName}}'s website.</p>
+  <p ng-show="officialInfo.website">Click <a href={{officialInfo.website}}>
+      here </a>to be redirected to {{officialInfo.candidateName}}'s website.
```
Comment (suparnabarua) [inline project/src/main/webapp/candidate-page/candidate-page.template.html]: Can you change this to "Candidate's website" and add the hyperlink to the website?

**Review by suparnabarua — APPROVED** [review]

**Review by kellyktan — APPROVED** [review]

File: project/src/main/webapp/navigation/header.component.js [hunk 2]
```diff
@@ -16,9 +16,14 @@
 "use strict"
 
 angular.module('navigation').component('header', {
-  template: `<div class="py-2 mb-3" id="site-header">
-              <h1>Voter Central</h1>
-             </div>`,
+  template: `<a href="/">
+              <div class="py-2 mb-3" id="site-header">
```
Comment (KathyFeiyang) [inline project/src/main/webapp/navigation/header.component.js]: extra small nit: 2 indentations between the `a` and `div` elements

**Review by KathyFeiyang — APPROVED** [review]
LGTM!

File: project/src/main/webapp/candidate-page/candidate-page.template.html (hunk 1 above)
Comment (ganewman) [reply project/src/main/webapp/candidate-page/candidate-page.template.html]: Definitely. I can do that pretty easily.

**Review by tjyoung99 — APPROVED** [review]

**Pull request merged**